src/components/picker/index.js

```diff
@@ -144,7 +144,11 @@ class Picker extends Component {
     /**
      * Pass props to the list component that wraps the picker options (allows to control FlatList behavior)
      */
-    listProps: PropTypes.object
+    listProps: PropTypes.object,
+    /**
+     * Pass a custom background color to the modal picker.
+     */
+    overlayBackgroundColor: PropTypes.string
```

Comment:
I'd go with a more general solution and allow passing `pickerModalProps` 
This way, in the future, if there are other props users want to pass they will be able to..